plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/interactive/layered/LayeredConstraintReevaluation.xtend

```diff
@@ -0,0 +1,316 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ * 
+ * http://rtsys.informatik.uni-kiel.de/kieler
+ * 
+ * Copyright 2019 by
+ * + Kiel University
+ *   + Department of Computer Science
+ *     + Real-Time and Embedded Systems Group
+ * 
+ * This code is provided under the terms of the Eclipse Public License (EPL).
+ */
+package de.cau.cs.kieler.klighd.lsp.interactive.layered
+
+import de.cau.cs.kieler.klighd.kgraph.KNode
+import de.cau.cs.kieler.klighd.lsp.interactive.ConstraintProperty
+import de.cau.cs.kieler.klighd.lsp.interactive.InteractiveUtil
+import java.util.HashSet
+import java.util.LinkedList
+import java.util.List
+import org.eclipse.elk.alg.layered.options.LayeredOptions
+
+/**
+ * @author cos, sdo
+ * 
+ */
+class LayeredConstraintReevaluation {
+
+    LinkedList<ConstraintProperty> changedNodes = newLinkedList()
+    KNode target
```

Comment:
Javadoc for these variables. Also please use @Accessors(PUBLIC_GETTER) instead of writing down the getters.

Reply:
Ok.